tests-behat/bootstrap/Context.php

```diff
@@ -428,6 +429,86 @@ public function iClickIconUsingCss($arg1)
         $icon->click();
     }
 
+    //// ScopeBuilder rule /////////
+
+    /**
+     * @Then /^text rule "([^"]*)" operator is "([^"]*)" and value is "([^"]*)"$/
+     */
+    public function textRuleOperatorAndValue($arg1, $arg2, $arg3)
+    {
+        $rule = $this->assertRuleExist($arg1);
+        $this->assertRuleOperatorSelectedValue($rule, $arg2);
+        $this->assertRuleInputValue($rule, $arg3);
+    }
+
+    /**
+     * @Then /^hasRef rule "([^"]*)" operator is "([^"]*)" and value is "([^"]*)"$/
```

Comment:
Then ScopeBuilder rule...

and name method like scopeBuilderHasRef...

and not need for comment `//// ScopeBuilder rule /////////` above